articles/libraries/lock-android/v1/native-social-authentication.md

```diff
@@ -2,7 +2,7 @@
 toc_title: Native Social Authentication
 description: How to implement native social authentication with Lock Android
 ---
-# Lock Android: Native Social Authentication
+# Native Social Authentication
```

Comment:
Again pointing out the thing with the headers. Here you don't have either a version or any mention of `Lock Android`.